jaeger-core/src/test/java/com/uber/jaeger/TracerTest.java

```diff
@@ -66,6 +68,26 @@ public void testBuildSpan() {
   }
 
   @Test
+  public void testBuildServerSpan() {
+    Span span = (com.uber.jaeger.Span) tracer.buildSpan("flexo")
+                                             .withTag(Tags.SPAN_KIND.getKey(), Tags.SPAN_KIND_SERVER)
```

Comment:
This is what it looks like on my IDE, I think it is reasonable indenting. Putting `.buildSpan` on the next line would simply move the block to the left by a few characters. Could you elaborate on why the current indenting is insane? Perhaps github is rendering things differently?
![image](https://cloud.githubusercontent.com/assets/833009/20820594/532f8714-b80a-11e6-832f-be5e21a0b060.png)